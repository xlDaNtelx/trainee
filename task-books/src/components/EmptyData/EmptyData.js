import React from 'react';
import { Alert } from 'reactstrap';

class EmptyData extends React.Component {
  render() {
    return (
      <Alert color="warning">
        Sorry there are no books, happens with the best of us.
      </Alert>
    );
  }
}

export default EmptyData;
