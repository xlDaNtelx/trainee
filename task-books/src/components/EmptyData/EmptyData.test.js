import React from 'react';
import renderer from 'react-test-renderer';
import EmptyData from './index';

test('EmptyData snapshot test', () => {
  const component = renderer.create(
    <EmptyData />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
