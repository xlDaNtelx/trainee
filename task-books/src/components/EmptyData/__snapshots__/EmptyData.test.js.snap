// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyData snapshot test 1`] = `
<div
  className="alert alert-warning fade"
  role="alert"
>
  Sorry there are no books, happens with the best of us.
</div>
`;
